AV-101710 - Documentation update all resources (#331)
couchbasecloud/terraform-provider-couchbase-capella: docs/resources/cluster_onoff_ondemand.md

page_title: "couchbase-capella_cluster_onoff_ondemand Resource - terraform-provider-couchbase-capella"
subcategory: ""
description: |-

>   Manages the On/Off state of an operational cluster. TThis resource is used to turn your cluster on or off.

Reviewer: ```suggestion
Manages the On/Off state of an operational cluster. This resource allows you to turn your cluster on or off.
```
